ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -278,18 +282,19 @@ public void ExportToFile(string resourceName, string fileName, VModel model, Can
             }
 
             DstDir = Path.GetDirectoryName(fileName);
-
             var exportedModel = CreateModelRoot(resourceName, out var scene);
+            var externalRefsHandled = new HashSet<string>();
 
             // Add meshes and their skeletons
             var loadedMeshDictionary = new Dictionary<string, Mesh>();
-            LoadModel(exportedModel, scene, model, resourceName, Matrix4x4.Identity, loadedMeshDictionary);
+            LoadModel(exportedModel, scene, model, resourceName, Matrix4x4.Identity, loadedMeshDictionary, externalRefsHandled);
 
             WriteModelFile(exportedModel, fileName);
+            return externalRefsHandled;
         }
 
         private void LoadModel(ModelRoot exportedModel, Scene scene, VModel model, string name,
-            Matrix4x4 transform, IDictionary<string, Mesh> loadedMeshDictionary, string skinName = null)
+            Matrix4x4 transform, IDictionary<string, Mesh> loadedMeshDictionary, HashSet<string> externalRefsHandled, string skinName = null)
         {
```

Comment:
The method handles vanim_c and vagrp_c too but `model.GetAllAnimations(FileLoader)` hides their filenames out.